internal/dbutil/pgutil/schema.go

```diff
@@ -8,18 +8,46 @@ import (
 	"crypto/rand"
 	"database/sql"
 	"encoding/hex"
+	"io"
 	"net/url"
 	"strconv"
 	"strings"
 )
 
-// RandomString creates a random safe string
-func RandomString(n int) string {
+const (
+	randRetries = 2
+)
+
+// RandomStringFromReader creates a random safe string sourced from the specified reader.
+func RandomStringFromReader(n int, r io.Reader) string {
```

Comment:
(a) If this is meant to be of use all throughout the codebase, we should probably put it somewhere more general under `internal/`. Maybe start a new package called internal/rand or internal/ioutils or something? If this is only meant to be of use here in this package, we should probably not export it.

Reply:
Good idea! I'll see if there are any other uses and refactor if I find some.

Reply:
(b) there is no (b). I don't know why I put (a) at the beginning of that comment.

Reply:
All other places should have proper error handling appropriate for the exact location.